README.md

```diff
@@ -17,7 +17,7 @@ for more details.
 ## Project Status
 Status: Alpha
 
-Latest image: `gcr.io/google-containers/volume-csi/gcp-filestore-csi-driver:v0.2.0`
+Latest image: `gcr.io/gke-release/gcp-filestore-csi-driver:v0.2.0`
```

Comment:
can use: `gke.gcr.io`

"The vanity domain 'gke.gcr.io' provides the closest mirror available for the request. If the request is from EU it should serve an image from 'eu.gcr.io/gke-release/'."